stdlib/public/core/StringStorage.swift

```diff
@@ -698,7 +730,9 @@ final internal class __SharedStringStorage
   }
 
   @inline(__always)
-  final internal var isASCII: Bool { return _countAndFlags.isASCII }
+  final internal var isASCII: Bool {
+    return _countAndFlags.isASCII
+  }
 
```

Comment:
Why this diff?

Reply:
I think it's a leftover that used to do something useful but doesn't due to master changing since then, I can remove it